### ISSUE TITLE
@SuppressWarnings('unused') doesn't work

### ISSUE THREAD

**Issue by mlewis-everley** [phpmd/phpmd#600]
I have followed the docs and believe this is setup correctly. I am using PHPMD 2.6.0. Basically I have the following class:

```php
class ContractAdmin extends ModelAdmin
{
    /**
     * @SuppressWarnings('unused')
     */
    private static $menu_priority = 20;
}
```
I have also tried:

```php
class ContractAdmin extends ModelAdmin
{
    /**
     * @SuppressWarnings(PHPMD.UnusedPrivateField)
     */
    private static $menu_priority = 20;
}
```
And finally:

```php
class ContractAdmin extends ModelAdmin
{
    /**
     * @SuppressWarnings(PHPMD)
     */
    private static $menu_priority = 20;
}
```
But when I run:

`./vendor/bin/phpmd app/src/Admin/ContractAdmin.php text unusedcode`

I still get:

`app/src/Admin/ContractAdmin.php:52    Avoid unused private fields such as '$menu_priority'.`

I have seen other issues reporting `@SuppressWarnings` as not working correctly, but none were about unusedcode (hence I have created a separate issue).

**Comment by tvbeek** [comment]
@mlewis-everley Thanks for reporting, I have a small question. Can you do 2 things for me?

1. Verify if you got the same problem with PHPMD 2.7
2. And if you still get it, can you try what happen if you add the `@SuppressWarnings` to the class docblock?

The first can help is in preventing searching for a bug that is maybe already fixed. The second can help us to find out if it totally didn't work or only not on docblocks for a variable.

**Comment by kylekatarnls** [comment]
I confirm this happens in PHPMD 2.7.0 and I confirm both `@SuppressWarnings(unused)` and `@SuppressWarnings(PHPMD.UnusedPrivateField)` correctly suppress the warning it put on the class doc block but not at property level.

**Comment by AJenbo** [comment]
This has been deprecated in favor of:
```php
use PHPMD\Rule\;
#[CoversClass(UnusedPrivateField::class)]
```
